pseudocode.txt

```diff
@@ -6,8 +6,8 @@ TCGA VCF file Subset into breast, ovarian, colo-rectal:
 
 Rows:
     If chrom in range of genes in pathway
-    If pos in range of genes in pathway
-        Include the row in new file
+    	If pos in range of genes in pathway
```

Comment:
This might be old, but here's where you need to test the chrom and pos together, with an and clause, since teh location ranges you're looking for are defined by the combination of chrom and pos together.